sdk/data/azappconfig/response_types.go

```diff
@@ -78,3 +79,70 @@ type SetSettingResponse struct {
 	// SyncToken contains the value returned in the Sync-Token header.
 	SyncToken SyncToken
 }
+type ArchiveSnapshotResponse struct {
+	generated.Snapshot
+
+	// Link contains the information returned from the Link header response.
+	Link *string
```

Comment:
What's in the link header?

Reply:
something in this format:

"</kv?snapshot=<SnapshotName>&api-version=2023-10-01>; rel=\"items\""